### ISSUE TITLE
Windows - sensu-client.log does not rotate

### ISSUE THREAD

**Issue by dzeleski** [sensu/sensu-puppet#618]
## Description of problem

Sensu wrapper service is not configured to rotate logs on windows

## Platform and version information

Occurs on Windows Server 2012R2

## Info

erb template that causes the issue: https://github.com/sensu/sensu-puppet/blob/master/templates/sensu-client.erb

winsw info around logging: https://github.com/kohsuke/winsw/blob/master/doc/loggingAndErrorReporting.md#rotate-mode

Im thinking we add the following to the erb template, this would keep 10x 10MB log files and delete the oldest.
```
<log mode="roll-by-size">
      <sizeThreshold>10240</sizeThreshold>
      <keepFiles>10</keepFiles>
</log>
```

**Comment by dzeleski** [comment]
Adding the logpath option as well:
```
  <logpath>C:\opt\sensu\</logpath>
  <log mode="roll-by-size">
        <sizeThreshold>10240</sizeThreshold>
        <keepFiles>10</keepFiles>
</log>
```

Going to let this burn for a few days and we will see what happens.

**Comment by jothoma1** [comment]
Hi, can you explain where do you put these options ?

**Comment by dzeleski** [comment]
@jothoma1 sure: `C:\opt\sensu\bin\sensu-client.xml` is where it gets added. Youll also need to remove the -l option off of the arguements, if you click on the pull request above you can see the changes that were made.

**Comment by ghoneycutt** [comment]
Thank you @dzeleski for opening this issue and @jothoma1 for your help.

This appears to be solved by PR #621 

Please re-open if this is not the case.
